openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -376,6 +427,29 @@ def _post_body(self, user, client, token_type=None, scope=None, asymmetric_jwt=N
 
         return body
 
+    def _test_jwt_access_token(self, client_attr, token_type=None, headers=None, grant_type=None, asymmetric_jwt=False):
```

Comment:
Nit: I like `_check` or `_verify` or `_assert_` or anything other than `_test_`, to make it clear it isn't a test.

Reply:
Used `test_` in consistency with [other already existing code in the same file](https://github.com/openedx/edx-platform/blob/062a7602427c4292964b444363e7ae5363654c6f/openedx/core/djangoapps/oauth_dispatch/tests/test_views.py#L178)